package.json

```
@@ -1,9 +1,31 @@
 {
+  "name": "sd-03-week3-html-css-project",
+  "version": "1.0.0",
+  "description": "Você já usa o GitHub diariamente para desenvolver os exercícios, certo? Agora, para desenvolver os projetos, você deverá seguir as instruções a seguir. Fique atento a cada passo, e se tiver qualquer dúvida, nos envie por _Slack_! #vqv 🚀",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Aqui no script não teria que especificar que ele roda os tests com o jest?

```suggestion
    "test": "jest",
```